tests/FixedArrayTest.php

```diff
@@ -166,6 +166,64 @@ public function providerShiftDown() : array
         ];
     }
 
+    public function testCreate()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+        $result = $fixedArray->create();
```

Comment:
`create()` was meant to be a static method, I just fixed it!
Can you please rebase your commits, and rewrite the test to check that it creates an array of the given size?

Reply:
I will pull your original repo and fix the test!